### DISCUSSION TITLE
Windows version?

### DISCUSSION THREAD

**Discussion by MA-MacDonald** [grafana/mimir#1763] [General]
Is there a Windows version of Mimir or will there be one in the future?

**Comment by pstibrany** [comment]
Hello,

we don't currently build Windows version of Mimir. We haven't really discussed if we want to do that, so I will not comment on future plans just yet. In my opinion, if the binary compiles, we could publish .exe file in the next release, with caveat that it's untested. But let's see what others think.

**Reply by pracucci** [reply]
I have mixed feelings. I'm not sure we should really commit to release Mimir for Windows, given our inability to properly test it release after release. So far we just committed to release `mimirtool` for Windows and I would get stick to it, unless we see a significant interest releasing Mimir for Windows. That being said, in the short term, community users can still compile Mimir for Windows themselves and submit patches if anything doesn't work as expected on Windows.

**Comment by aknuds1** [comment]
I already fixed some network/filesystem path separator [inconsistencies](https://github.com/thanos-io/thanos/pull/5281) in Thanos. That makes me think that support for Windows would require examining the codebase for similar issues, and we should probably also continuously test on Windows to avoid regressions.
